### ISSUE TITLE
ask code interactively

### ISSUE THREAD

**Issue by amkartashov** [gevial/mfaprof#1]
Thanks for the tool. Can you add functionality to ask code interactively with terminal prompt? For now invocations of mfaprof are spamming bash history because of different codes.

W/A I use for now is bash alias:

```bash
$ type mfaprof_i
mfaprof_i is aliased to `mfaprof -code $(read -r -p"code: " code; echo $code)'

$ mfaprof_i -profile ...
code: 123456
2023/10/26 10:48:48 No MFA device serial number provided, getting one from ListMFADevices
2023/10/26 10:48:49 Getting temporary credentials
2023/10/26 10:48:49 Saving new profile
...
```

**Comment by gevial** [comment]
Excellent idea, thanks! I’ll add it. Will try to finish in two weeks time.

**Comment by gevial** [comment]
Addressed in the v1.1.0 release. Just omit `-code` and the tool will ask interactively.
